### PR TITLE
Implement a ban command, and wording

### PR DESCRIPTION
**Pull request by Hadock-is-ok**
**-** Add new ban **command**
**-** fix wording in pits.py
**-** change a id usage to a constant usage

### CODE REVIEW

File: cogs/hideout/moderation.py [hunk 1]
```diff
@@ -21,15 +21,21 @@ async def ban(self, ctx: HideoutContext, member: discord.Member, *, reason: Unti
...
-        if time:
-            await self.bot.create_timer(time.dt, 'tempban', ctx.guild.id, member.id, precise=False)
-            fmt = f"until {discord.utils.format_dt(time.dt, 'R')}"
+        if reason:
+            if reason.value:
+                fmt = f" for {reason.value}"
+            else:
+                fmt = ""
+            time = reason.flags._for
+            print(time)
```
Comment (crygup) [inline cogs/hideout/moderation.py]: Remove debug print

Comment (Hadock-is-ok) [reply cogs/hideout/moderation.py]: done

File: cogs/hideout/moderation.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            time = reason.flags._for
+            if time:
+                #await self.bot.create_timer(time.dt, 'tempban', ctx.guild.id, member.id, precise=False)
+                fmt += f"until {discord.utils.format_dt(time.dt, 'R')}"
+        else:
+            fmt = ""
+
+        try:
+            await member.send(f"You have been banned from Duck Hideout{fmt}.")
+        except (discord.HTTPException, discord.Forbidden):
```
Comment (LeoCx1000) [inline cogs/hideout/moderation.py]: Forbidden inherits HTTPException
```suggestion
        except discord.HTTPException:
```

File: cogs/hideout/moderation.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                fmt = ""
+            time = reason.flags._for
+            if time:
+                #await self.bot.create_timer(time.dt, 'tempban', ctx.guild.id, member.id, precise=False)
+                fmt += f"until {discord.utils.format_dt(time.dt, 'R')}"
+        else:
+            fmt = ""
+
+        try:
+            await member.send(f"You have been banned from Duck Hideout{fmt}.")
```
Comment (LeoCx1000) [inline cogs/hideout/moderation.py]:
```suggestion
            await member.send(f"You have been banned from Duck Hideout{fmt}.\nAppeal this action by emailing `support@duck-bot.com`")
```

File: cogs/hideout/moderation.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                #await self.bot.create_timer(time.dt, 'tempban', ctx.guild.id, member.id, precise=False)
+                fmt += f"until {discord.utils.format_dt(time.dt, 'R')}"
+        else:
+            fmt = ""
+
+        try:
+            await member.send(f"You have been banned from Duck Hideout{fmt}.\nAppeal this action by emailing `support@duck-bot.com`")
+        except discord.HTTPException:
+            pass
+
```
Comment (LeoCx1000) [inline cogs/hideout/moderation.py]:
```suggestion
    await member.ban(reason=f"banned by {ctx.author}{fmt}."[:500])
```

**Pull request merged**